src/renderer/components/dashboardPage/issues/form/index.vue

```diff
@@ -200,11 +208,21 @@ export default {
         message: 'Issue saved',
         type: 'success'
       })
-      this.closeDialog()
+      if (this.createAnother) {
+        this.openDialog({
+          ...this.form,
+          summary: '',
+          description: ''
+        })
+        this.$el.querySelector('.form-wrapper').scrollTop = 0
```

Comment:
Use a [ref](https://vuejs.org/v2/api/#ref) instead of searching the DOM

Reply:
Yes, my bad. Thank you :)